File: player/src/main/java/com/we/player/player/PlayerEventListener.java
package com.we.player.player;

/**
 * @Description:
 * @Author: Wisn
 * @CreateDate: 2020/11/14 上午9:06
 */
public interface PlayerEventListener {
    void onPlayerEventError();

    void onPlayerEventCompletion();

    void onPlayerEventInfo(int what, int extra);

    void onPlayerEventPrepared();

    void onPlayerEventVideoSizeChanged(int width, int height);
}

File: player/src/main/java/com/we/player/player/PlayerEventListener.kt
package com.we.player.player;

/**
 * @Description:
 * @Author: Wisn
 * @CreateDate: 2020/11/14 上午9:06
 */
interface PlayerEventListener {
    fun onPlayerEventError();

    fun onPlayerException(message: String?);

    fun onPlayerEventCompletion();

    fun onPlayerEventInfo(what: Int, extra: Int);

    fun onPlayerEventPrepared();

    fun onPlayerEventVideoSizeChanged(width: Int, height: Int);
}
